api/resource/resource.proto

```diff
@@ -11,6 +11,8 @@ import "google/protobuf/timestamp.proto";
 //
 // ResourceService provides user-facing API for the Talos resources.
 service ResourceService {
+  option (common.remove_deprecated_service) = "v1.5";
```

Comment:
old Talos resources API is now deprecated, but fully supported until 1.5

new `talosctl` is using only COSI API